src/framework/audio/internal/encoders/oggencoder.cpp

```diff
@@ -22,7 +22,11 @@
 
 #include "oggencoder.h"
 
+#ifdef SYSTEM_OPUSENC
+#include "opus/opusenc.h"
```

Comment:
Wouldn’t it be less confusing to use `<>` for including system header files?

```suggestion
#include <opus/opusenc.h>
```

Reply:
> Wouldn’t it be less confusing to use `<>` for including system header files?

This worked, also allowing the `set(MODULE_INCLUDE` to be removed. Thanks!